extensions/google-drive/src/utils.ts

```diff
@@ -161,6 +164,33 @@ ${new Date(file.createdAt).toLocaleString()}
 ${new Date(file.updatedAt).toLocaleString()}`;
 };
 
+export const fileStream = ({ stats = false }: { stats?: boolean } = {}) => {
```

Comment:
```suggestion
type FileStreamOptions = { stats?: boolean };
export const fileStream = ({ stats }: FileStreamOptions = {}): Transform => {
```

for better readability